install/src/Env.hs

```diff
@@ -54,7 +54,7 @@ findInstalledGhcs = do
     -- sort by version to make it coherent with getHlsVersions
     $ sortBy (comparing fst)
     -- nub by version. knownGhcs takes precedence.
-    $ nubBy ((==) `on` fst)
+    $ nubOrdBy (compare `on` fst)
```

Comment:
Maybe this change has broken the install script??? It shouldn't but is the unique direct change in that component. 
@jhrcek please, could you try the install reverting this change manually?

Reply:
If the change broke it, hlint suggestion was very wrong :thinking:

Reply:
@jneira Just tried with latest master and I had to revert both this change (nubOrdBy back to nubBy) AND one more change done in `install/src/Print.hs` below. After reverting those two changes it started working again.

Reply:
I don't know if there's some kind of implicit import going on or something ,but those functions come from `extra` and I don't see them imported in either of the 2 modules.

Reply:
No implicit imports afaik, fortunately :slightly_smiling_face: 
The fix would be add those imports probably

Reply:
Sure, I'll do it later today.

Reply:
I guess this part of the code isn't tested by the CI then? Sorry about that.

@jneira thanks for the merge!
@jhrcek thanks for fixing it!

(I did not except this PR to be so fiddly and hard to merge.)

Reply:
yeah, the install script is not being tested, but it does not change frequently so 🤷